.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,112 @@
+name: Continuous integration
+on: [push]
+env:
+  TOX_PARALLEL_NO_SPINNER: 1
+  PYTEST_ADDOPTS: -x
```

Comment:
I tend prefer to longhand options (`--exitfirst`) in code except for very well known ones, as this makes the meaning more obvious and shorthands can be hard to search for.

Reply:
Done